src/components/DropdownItem/DropdownItem.tsx

```diff
@@ -0,0 +1,41 @@
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  person: Person;
+  onSelectedPerson: (person: Person) => void;
+  onQuerry: (value: string) => void;
```

Comment:
Try to correct all the typos in the code.
```suggestion
  onQuery: (value: string) => void;
```